ext/nnstreamer/tensor_filter/tensor_filter_deepview_rt.cc

```diff
@@ -0,0 +1,683 @@
+/**
+ * GStreamer Tensor_Filter, DeepViewRT Module
+ * Copyright 2022 NXP
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Library General Public
+ * License as published by the Free Software Foundation;
+ * version 2.1 of the License.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Library General Public License for more details.
+ *
+ */
```

Comment:
If you subplugin uses C++, it is recommended to inherit ```tensor_filter_subplugin``` class of ```<nnstreamer_cppplugin_api_filter.hh>```; it's easier to write and maintain.

However, using C subplugin struct still works and it's fine to do so.

Reply:
Example of using C++ class for tensor-filter subplugin: 

https://github.com/nnstreamer/nnstreamer/blob/main/ext/nnstreamer/tensor_filter/tensor_filter_edgetpu.cc

https://github.com/nnstreamer/nnstreamer/blob/main/ext/nnstreamer/tensor_filter/tensor_filter_mediapipe.cc

https://github.com/nnstreamer/nnstreamer/blob/main/ext/nnstreamer/tensor_filter/tensor_filter_snpe.cc

Reply:
updated in last patchset: subplugin defined subclassing tensor_filter_subplugin

Reply:
Great! Thanks!!